bin/ssh_scan

```diff
@@ -142,6 +142,11 @@ opt_parser = OptionParser.new do |opts|
     exit
   end
 
+  opts.on("-l", "--listen", "Listen and serve API requests") do
+    SSHScan::API.run!
+    exit
+  end
```

Comment:
would also be nice to select the bind port for the API in the new binary, so it can be run on 443 in prod.